svg/_types.py

```diff
@@ -41,10 +41,10 @@ def __str__(self) -> str:
 
 @dataclass
 class ViewBoxSpec:
-    min_x: int
-    min_y: int
-    width: int
-    height: int
+    min_x: float
+    min_y: float
+    width: float
+    height: float
 
     def __str__(self) -> str:
-        return f"{self.min_x} {self.min_y} {self.width} {self.height}"
+        return f"{self.min_x:g} {self.min_y:g} {self.width:g} {self.height:g}"
```

Comment:
What's the motivation for the change? `g` turns into scientific notation after 6 signs:

```
'{:g}'.format(10 ** 6)  # '1e+06'
```

Does SVG standard allow scientific notation?

Reply:
The motivation was actually to prevent something like `0 0 100 100` turning into `0.0 0.0 100.0 100.0` though looking at the spec again (and the MDN one actually being a direct quote of this) it does seem to allow for scientific notation as well. 

https://www.w3.org/TR/SVG11/types.html#DataTypeNumber

>When used in an SVG attribute, a `<number>` is defined differently, to allow numbers with large magnitudes to be specified more concisely:
>
>     number ::= [integer](https://www.w3.org/TR/SVG11/types.html#Integer) ([Ee] [integer](https://www.w3.org/TR/SVG11/types.html#DataTypeInteger))?
>                | [+-]? [0-9]* "." [0-9]+ ([Ee] [integer](https://www.w3.org/TR/SVG11/types.html#DataTypeInteger))?

Reply:
If you don't specify any specifier (as it used to be), `int` will be represented without zeros and `float` with. Maybe, it would be great to drop zeros from float one day, but lets for now keep it consistent with the rest of the code. Currently, all numbers are converted by just calling `str` on them: https://github.com/orsinium-labs/svg.py/blob/master/svg/elements.py#L45. Still, great idea to look into it, thanks 👍🏿